tests/ccl_test_growth.py

```diff
@@ -26,7 +26,7 @@ def read_growth_lowz_benchmark_file():
     """
     # Load data from file
     dat = np.genfromtxt(join(dirname(__file__),"benchmark/growth_model1-5.txt")).T
-    assert(dat.shape == (6,6))
+    assert_(dat.shape == (6,6))
```

Comment:
Ditto here.

Reply:
Yeah so assert is not a function in python 3 and the rest of the code used `assert_`. I was trying to stick to one thing. Happy to change it back. 🤷‍♀️

Reply:
It's not a function, but `assert(a==b)` still works, so you don't need to care about that if you like the parentheses.